hack/verify-all-gensrc.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs all of the Go source verification scripts inside of the
+# ./hack/go-tools directory. The success or failure of each script is outputted
+# in green or red colored text, respectively. If any script fails, an error is
+# returned, otherwise returns 0.
+
+set -euo pipefail
+
+# TODO: Run `kubebuilder generate` and verify that nothing has changed.
```

Comment:
We'll also need to run `go vet` as well.

Reply:
Added a note.